taskcluster/focusios_taskgraph/__init__.py

```diff
@@ -0,0 +1,27 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
This is a copy of https://github.com/mozilla-mobile/firefox-ios/blob/main/taskcluster/ffios_taskgraph/__init__.py

Folder renamed, instead of `ffios_taskgraph` it is now `focusios_taskgraph`